hops/featurestore_impl/rest/rest_rpc.py

```diff
@@ -219,3 +219,108 @@ def _put_trainingdataset_create_job(job_conf):
                     constants.DELIMITERS.SLASH_DELIMITER +
                     constants.REST_CONFIG.HOPSWORKS_TRAININGDATASETS_CREATION_RESOURCE)
     return _http(resource_url, method = constants.HTTP_CONFIG.HTTP_POST, headers=headers, data=job_conf)
+
+
+def _add_metadata(featurestore_id, featuregroup_id, name, value):
+    """
+    Makes a REST call to Hopsworks to attach extended metadata to a featuregroup
+
+    Args:
+        :featurestore_id: the id of the featurestore
+        :featuregroup_id: the id of the featuregroup
+        :name: the name of the extended attribute
+        :value: the value of the extended attribute
+
+    Returns:
+        None
+
+    Raises:
+        :RestAPIError: if there was an error in the REST call to Hopsworks
+    """
+    headers = {constants.HTTP_CONFIG.HTTP_CONTENT_TYPE: constants.HTTP_CONFIG.HTTP_APPLICATION_JSON}
+    data = json.dumps({name: str(value)})
+    resource_url = (constants.DELIMITERS.SLASH_DELIMITER +
```

Comment:
Can you use the function _get_api_featurestore_path_id() in this module to make this more readable.

Reply:
Ok